sims/check.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf-8
+import msprime
+import random
+from matplotlib import pyplot as plt
+
+
+random.seed(111)
+ft_ts = msprime.load('simupoptree.hdf5')
+L = ft_ts.sequence_length
+fwd_ts = msprime.load('fwdpptree.hdf5')
+L = ft_ts.sequence_length
+#assert L == fwd_ts.sequence_length, "fwdpp {} and simupop {} don't match".format(fwd_ts.sequence_length, L)
+NUM = 100
+
+tss = {'simupop': ft_ts, 'fwdpp':  fwd_ts}
+
+L = fwd_ts.sequence_length
+theta = 10.
+rho = 10.
+mut_rate = theta/4./ L
```

Comment:
You need to divide by Ne here as well for the various rates.

Reply:
Ah, that may well be the issue when comparing ftprime/simupop and msprime. Otherwise I think msprime and ftprime are in similar units